src/renderers/shared/fiber/isomorphic/ReactCoroutine.js

```diff
@@ -42,7 +42,9 @@ export type ReactYield = {
   continuation: mixed
 };
 
-exports.createCoroutine = function<T>(children : mixed, handler : CoroutineHandler<T>, props : T, key : ?string = null) : ReactCoroutine {
+exports.createCoroutine = function<T>(
```

Comment:
We generally wrap long arg lists [like this](https://github.com/lexjacobs/react/blob/7b47ef04ddd50ed70a11c1bc6363f1484c60ff08/src/renderers/shared/stack/reconciler/ReactCompositeComponent.js#L401-L407), can you tweak it?